index.js

```diff
@@ -7,12 +10,28 @@ function ask(questionText) {
   });
 }
 
+//----------------async code---------------//
+
 start();
 
 async function start() {
+  let max = 100
+  let min = 1
+  let compNumber = 50
   console.log("Let's play a game where you (human) make up a number and I (computer) try to guess it.")
   let secretNumber = await ask("What is your secret number?\nI won't peek, I promise...\n");
   console.log('You entered: ' + secretNumber);
-  // Now try and complete the program.
+  let firstGuess = await ask(`Is your number ${compNumber}?`)
+  if (firstGuess === 'yes') {
+    console.log('You got it!')
+  } else if (firstGuess === 'no') {
+    await ask('Is it higher or lower?')
+  }
```

Comment:
You are correct you will need a function to handle if the computer needs to guess higher or lower. This will probably look like a block of if statements, perhaps `if (answer === "h") {//code} else if (answer === "l"){//code}`